code/modules/events/antagonist_critter_ghost_respawn.dm

```diff
@@ -18,15 +20,34 @@
 		if (..())
 			return
 
-		var/input = input(usr,"Which one? Pick null for random???",src.name) as null|anything in list("fire_elemental","spider","gunbot")
-		if (!input || !istext(input))
+		var/input = input(usr,"Which one? Press cancel for random.",src.name) as null|anything in list("fire_elemental","spider","gunbot","custom critter path")
```

Comment:
you can outright remove the options here, just let them choose between custom and random